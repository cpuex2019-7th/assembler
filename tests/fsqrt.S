    li t1, 1035489772           ; 0.09
    fmvwx ft1, t1

    fsqrt ft0, ft1
    
    fmvxw a0, ft0               ; 0.3
