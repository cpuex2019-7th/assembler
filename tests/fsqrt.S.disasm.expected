lui t1, 252805
addi t1, t1, 492
fmvwx ft1, t1
fsqrt ft0, ft1
fmvxw a0, ft0
